data/themes/darktable.css

```diff
@@ -753,47 +755,61 @@ overshoot.right
 #main-histogram .toggle
 {
   border: 1.1px solid alpha(@button_bg, 0.3);
```

Comment:
This "odd" 1.1px border and color are to match the legacy look. Should we just stick with the "normal" theme-ing for these? See https://github.com/darktable-org/darktable/blob/c47cc5d54801ee3c08e3f242c3422db7c441ebf9/data/themes/darktable.css#L269-L282

Reply:
Do we really want legacy look? All UI since darktable 3 is quite different from legacy one on previous releases. I will test and check what we have without this border.

Reply:
I'd say we don't want to keep to the legacy look, but to respond to how the program looks and is used now. This legacy look will be increasingly jarring as the visuals of the rest of darktable shift, and won't make sense with the darker/dark/grey theme variants.

I kept close to "legacy" when I wrote this CSS, so that my work could be about a stable change to the underlying code. But the design work you're doing now is something which I was hoping that the code changes would allow.

All that said, I have a huge respect for the original implementation, and how some straightforward (non-CSS) code allowed for such a nice tool. And it takes jumping through a surprising number of hoops with GTK to give the nice buttons-appearing-on-mouseover behavior. It's another lesson for me in the ways that toolkits can be powerful, but also channel thinking.